app/routes/meetings/components/MeetingDetail.css

```diff
@@ -27,6 +26,12 @@
   justify-content: space-between;
 }
 
+.line {
+  border-bottom: 3px solid #e5e5e4;
```

Comment:
should use a color variable here that's used for other borders

Reply:
Deleted the line